test/commands/scanner/run.test.ts

```diff
@@ -895,50 +814,40 @@ describe('scanner:run', function () {
 					expect(uniqueEngines).to.contain("pmd");
 					// Validate that all of the results have an expected property
 					for (const result of results) {
-						expect(result.violations[0], `Message is ${result.violations[0].message}`).to.have.property("ruleName").that.is.not.null;
+						expect(result.violations[0], `Message is ${result.violations[0].message}\n ${ctx.stdout}`).to.have.property("ruleName").that.is.not.null;
 					}
 				});
 		});
 	});
 
-	it('BaseConfig Environment Tests For Javascript', () => {
-		test
-		.stdout()
-		.stderr()
-		.command(['scanner:run',
-			'--target', path.join('test', 'code-fixtures', 'projects', 'js', 'src', 'baseConfigEnv.js'),
-			'--format', 'csv'
-		])
-		.it('The baseConfig enables the usage of default Javascript Types', ctx => {
-			// There should be no violations.
-			expect(ctx.stdout).to.contains('No rule violations found.', 'Should be no violations found in the file.');
-		});
+	describe('BaseConfig Environment Tests For Javascript', () => {
+		setupCommandTest
+			.command(['scanner:run',
+				'--target', path.join('test', 'code-fixtures', 'projects', 'js', 'src', 'baseConfigEnv.js'),
+				'--format', 'csv'
+			])
+			.it('The baseConfig enables the usage of default Javascript Types', ctx => {
+				// There should be no violations.
+				expect(ctx.stdout).to.contains('No rule violations found.', 'Should be no violations found in the file.');
+			});
 
 		// TODO: THIS TEST WAS IMPLEMENTED FOR W-7791882. THE FIX FOR THAT BUG WAS SUB-OPTIMAL, AND WE NEED TO CHANGE IT IN 3.0.
 		//       DON'T BE AFRAID TO CHANGE/DELETE THIS TEST AT THAT POINT.
-		test
-			.stdout()
-			.stderr()
+		setupCommandTest
 			.command(['scanner:run',
 				'--target', path.join('test', 'code-fixtures', 'projects', 'js', 'src', 'fileThatUsesQUnit.js'),
 				'--format', 'json'
 			])
 			.it('By default, frameworks such as QUnit are not included in the baseConfig', ctx => {
 				// We expect there to be 2 errors about qunit-related syntax being undefined.
-				// There's currently some weird issue with the test framework that causes this specific execution to act
```

Comment:
This was a side effect of the --verbose test that was moved. Removed this special case code.